pkgs/development/tools/build-managers/conan/default.nix

```diff
@@ -16,16 +16,24 @@ let newPython = python.override {
         sha256 = "1080pdxrvnkr8i7b7bk0dfx6cwrkkzzfaranl7207q6rdybzqay3";
       };
     });
+    astroid = super.astroid.overridePythonAttrs (oldAttrs: rec {
+      version = "1.6.5";
+      disabled = false;
+      src = oldAttrs.src.override {
+        inherit version;
+        sha256 = "fc9b582dba0366e63540982c3944a9230cbc6f303641c51483fa547dcc22393a";
+      };
+    });
```

Comment:
@FRidh Do you have any idea why this doesn't work?

> astroid-2.0.1 not supported for interpreter python2.7

Reply:
Seems to work when I use `python3`. So, `packageOverrides` must be broken for `python2`.